packages/nc-gui-v2/pages/nc/[projectId]/index/index.vue

```diff
@@ -25,14 +29,32 @@ function openQuickImportDialog(type: string) {
   quickImportDialog.value = true
   importType.value = type
 }
+
+const icon = (tab: TabItem) => {
+  switch (tab.type) {
+    case TabType.TABLE:
+      return MdiAirTableIcon
+    case TabType.VIEW:
+      return MdiView
+    case TabType.AUTH:
+      return MdiAccountGroup
+  }
+}
 </script>
 
 <template>
   <div class="h-full w-full nc-container pt-[9px]">
     <div class="h-full w-full flex flex-col">
-      <div class="px-2">
+      <div class="">
```

Comment:
If this padding gets removed (remove the class="") then the `nc-grid-wrapper` should receive a `px-2`

Reply:
maybe even px-4 otherwise the grid is too close to both sidebars.